syntax = "proto3";

service FakePerson {
    rpc GetFakePersonData (ClientNameRequest) returns (FakePersonReply) {}
}

message ClientNameRequest {
    string name = 1;
}

message FakePersonReply {
    string name = 1;
    int32 height = 2;
}